pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -97,3 +141,19 @@ func readyProtocol(
 		return fmt.Errorf("unexpected context error: [%v]", ctx.Err())
 	}
 }
+
+func memberIDToAddress(
+	memberID MemberID,
+	publicKeyToAddressFn func(cecdsa.PublicKey) []byte,
+) (string, error) {
+	pubKey, err := memberID.PublicKey()
+	if err != nil {
+		return "", fmt.Errorf(
+			"could not get public key for member with ID [%s]: [%v]",
+			memberID,
+			err,
+		)
+	}
+
+	return "0x" + hex.EncodeToString(publicKeyToAddressFn(*pubKey)), nil
```

Comment:
nitpicking but I think it can be formatted like so: `return fmt.Sprintf("0x%v", hex.EncodeToString(publicKeyToAddressFn(*pubKey))), nil`